app/assets/javascripts/controllers/header/header_controller.js

```diff
@@ -47,6 +53,7 @@ angular.module('ManageIQ').controller('headerController', ['$scope', 'eventNotif
 
     vm.notificationsIndicatorTooltip = miqFormatNotification(__('%{count} unread notifications'),
       {count: notificationCount});
+      ManageIQ.redux.store.dispatch({ type: '@@notifications/setUnreadCount', payload: notificationCount.text });
```

Comment:
Could you please write a few words into the PR's description about this idea. I know how it works because I made it up :laughing: but others might not...